.gitignore

```diff
@@ -80,7 +80,7 @@ yarn-debug.log*
 /test/tmp/
 /test/version_tmp/
 /tmp/
-.idea/
+.idea/*
```

Comment:
You can follow this tutorial in order to remove everything within `./idea` from being tracked:
http://www.codeblocq.com/2016/01/Untrack-files-already-added-to-git-repository-based-on-gitignore/

Reply:
done.